content/docs/legal/privacy/website.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Website Privacy Policy
+toc_title: Website
+description: This page informs you of our policies regarding the collection, use, and disclosure of personal data when using the Scratch Addons website and the choices you have associated with that data.
+---
+
+> Effective date: 1 June 2021
+
+This page informs you of our policies regarding the collection, use, and disclosure of personal data when using the Scratch Addons website (<https://scratchaddons.com>) and the choices you have associated with that data.
+
+We use your data to provide and improve the website. By using the website, you agree to the collection and use of information in accordance with this policy.
+
+"Website" refers to to the Scratch Addons website (<https://scratchaddons.com>). This is the website you're on right now.
+
+## Summary
+
+- **We do not use cookies.**
+  
+  Local storage is used to store basic preferences such as whether dark mode is enabled.
+
+- **We track trends in website traffic, without tracking individual visitors.**
+  
+  We do not collect any personally identifiable information. You can see the same data we can see on the [public analytics dashboard](https://analytics.scratchaddons.com).
+
+- **You can send feedback anonymously.**
+  
+  Your IP address isn't stored when sending feedback.
+
+## Cookies and Local Storage
+
+We do not store cookies on your device.  
+We might, however, use local storage capabilities to store basic preferences such as whether dark mode is enabled on this website.
+
+## Analytics
+
+We use a self-hosted Plausible Analytics instance routed through our analytics subdomain to count website visits, downloads, etc. Plausible works without collecting any personal data or personally identifiable information, and without using cookies. You can see the same data we can see on the [public analytics dashboard](https://analytics.scratchaddons.com).
+
+## Feedback
+
+When sending feedback through <https://scratchaddons.com/feedback>, some data might be included in the request, such as:
+
+- Text written in the "identifiers" field, if any.
+- Text written in "feedback" field.
+- The version number for your Scratch Addons instance.
+- Your browser's user agent string (which often includes your browser's and operating system's names and version numbers).
+
+Feedback is sent to a public open source [Glitch.com Node.js server instance](https://glitch.com/edit/#!/scratchaddons-feedback) which then redirects the information to a [Discord](https://discord.com) server. We do not log the IP addresses of the users that send feedback, and you can verify that by looking at the Glitch server source code linked above.
```

Comment:
I've created a repo for this because it's not quite open source until people can report issues and PR. I've also taken the liberty of updating quite a few things, which are committed to the repo. I'll test these thoroughly. @WorldLanguages Is there a way you can pull from the repo to a (possibly new) glitch instance?

Reply:
> it's not quite open source until people can report issues and PR

I don't think that's true? Also it's a 100 line nodejs server come on

Reply:
> > it's not quite open source until people can report issues and PR
> 
> I don't think that's true? Also it's a 100 line nodejs server come on

It just needs an open source license.

Reply:
Lol maybe it shouldn't be "open source", more like "public source" :P

Reply:
But the repo exists (under a GPL license), so I think we're good?